pages/user.js

```diff
@@ -47,34 +47,41 @@ const SectionSwitch = ({ userId, section }) => {
     return <MailLog userId={userId} />
   }
   if (section === 'sessions') {
-    return <Fragment>
-      <Sessions userId={userId} />
-      <EventLog userId={userId} />
-    </Fragment>
+    return (
+      <Fragment>
```

Comment:
FYI you can also use the `<>...</>` syntax instead of `<Fragment>...</Fragment>`. Then you can spare yourself the Fragment import.

Reply:
Perfect, good to know. Since there are several places to change that, I've opened a small issue and we can do it whenever it fits